deployment/v3/utils/postgres-backup/delete.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Uninstalls postgres-backup
+## Usage: ./delete.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+function deleting_postgres-backup() {
+  NS=postgres-backup
+  while true; do
+      read -p "Are you sure you want to delete postgres-backup helm charts?(Y/n) " yn
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)